verify_test.go

```diff
@@ -342,6 +345,97 @@ func TestDistributedClaims(t *testing.T) {
 	}
 }
 
+func TestDistClaimResolver(t *testing.T) {
+	tests := []struct {
+		test    resolverTest
+		want    map[string]claimSource
+		wantErr bool
+	}{
+		{
+			test: resolverTest{
+				name:    "noAccessToken",
+				payload: `{"iss":"","aud":"client1","email":"janedoe@email.com"}`,
+				config: Config{
+					ClientID:        "client1",
+					SkipExpiryCheck: true,
+				},
+				signKey: newRSAKey(t),
+			},
+			want: map[string]claimSource{},
+		},
+		{
+			test: resolverTest{
+				name:    "rightAccessToken",
+				payload: `{"iss":"","aud":"client1","email":"janedoe@email.com"}`,
+				config: Config{
+					ClientID:        "client1",
+					SkipExpiryCheck: true,
+				},
+				signKey:     newRSAKey(t),
+				accessToken: "1234",
+			},
+			want: map[string]claimSource{},
+		},
+		{
+			test: resolverTest{
+				name:    "wrongAccessToken",
+				payload: `{"iss":"","aud":"client1","email":"janedoe@email.com"}`,
+				config: Config{
+					ClientID:        "client1",
+					SkipExpiryCheck: true,
+				},
+				signKey:     newRSAKey(t),
+				accessToken: "12345",
+			},
+			wantErr: true,
+		},
+	}
+	for _, test := range tests {
+
+		t.Run(test.test.name, func(t *testing.T) {
+
+			endpoint, err := test.test.testEndpoint(t)
```

Comment:
this doesn't return an endpoint? does it? Need to rename it as "claims"

Also please remove the whitespace in the two lines above.